scripts/quests/bastok/The_Elvaan_Goldsmith.lua

```diff
@@ -40,37 +41,41 @@ quest.sections =
         },
     },
 
+    -- These functions check the status of ~= QUEST_AVAILABLE to support repeating
+    -- the quest.  Does not have to be flagged again to complete an additional time.
     {
         check = function(player, status, vars)
-            return status == QUEST_ACCEPTED or
-                (
-                    status == QUEST_COMPLETED and
-                    player:getFameLevel(xi.quest.fame_area.BASTOK) == 1 and
-                    not quest:getMustZone(player)
-                )
+            return status ~= QUEST_AVAILABLE and
+            player:getFameLevel(xi.quest.fame_area.BASTOK) >= 1 or
```

Comment:
Shouldn't need to check for 1 fame. Players start with 1. Therefore, this will always return true regardless of the conditional statement below